modules/appData.js

```diff
@@ -19,7 +18,6 @@ function getDeploymentData() {
 
 const deploymentData = getDeploymentData();
 const environment = config.util.getEnv('NODE_ENV'); // Defaults to 'development'
-debug(`App environment is ${environment}`);
```

Comment:
Logging this in `bin/www` rather than is this file which gets imported everywhere as a general utility.